R/utilities-container.R

```diff
@@ -22,7 +26,26 @@ getAllContainersMatching <- function(path, container) {
   validateIsOfType(container, c("Simulation", "Container"))
   validateIsOfType(path, "character")

```

Comment:
I thought it is better to have one generic function that checks for a given class instead of multiple helper-functions (isNumeric etc). But sure, we can also define `isCharacter `and `isNumeric`.

Reply:
R has built-in functions `is.numeric()` and `is.character()`, what we could do is to pack them in `validateIsNumeric()` and `validateIsCharacter()`